src/pages/auth/SignUpConfirmation/SignUpConfirmation.tsx

```diff
@@ -15,32 +15,39 @@ const schema = object({
 });
 
 export const SignUpConfirmation = () => {
-  const [ searchParams ] = useSearchParams();
+  const [searchParams] = useSearchParams();
   const navigate = useNavigate();
-  const email = searchParams.get('email') || '';
+  const email = searchParams.get("email") || "";
   useEffect(() => {
     if (!email) {
-      navigate('/');
+      navigate("/");
     }
   }, [email, navigate]);
-  const { register, handleSubmit, formState: { errors } } = useForm<ConfirmationData>({
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm<ConfirmationData>({
     resolver: yupResolver(schema),
     defaultValues: {
-      code: ''
-    }
+      code: "",
+    },
   });
 
-  const mutation = useMutation((code: string) => {
-    return AuthService.confirmSignUp({ email, code });
-  }, {
-    onSuccess: () => {
-      navigate('/');
+  const mutation = useMutation(
+    (code: string) => {
+      return AuthService.confirmSignUp({ email, code });
     },
-  });
+    {
+      onSuccess: () => {
+        navigate("/");
+      },
+    }
+  );
 
   const onSubmit = handleSubmit(async ({ code }) => {
     try {
-      mutation.mutate(code)
+      mutation.mutate(code);
```

Comment:
Can you please add [semicolon rule](https://eslint.org/docs/latest/rules/semi) to eslint config?